packages/curve-common/src/widgets/Header/SideBarFooter.tsx

```diff
@@ -30,6 +31,7 @@ export const SideBarFooter = ({ themes: [theme, setTheme], advancedMode, WalletP
     >
       <Box display="flex" paddingX={4} marginTop={4}>
         <ConnectWalletIndicator {...WalletProps} sx={{ flexGrow: 1 }} />
+        <UserProfileButton />
```

Comment:
The theme switcher is not rendered in two places. The alignment here doesn't seem correct either
![image](https://github.com/user-attachments/assets/013b7adc-f823-4874-b713-585d2bfca487)
```suggestion
        {isBeta && <UserProfileButton />}
```

Reply:
Ho lol, a remainder from earlier experimentations. I've removed this button from the sidebar altogether, there's no need for it as there's already settings to change theme and advanced mode.